docrawl/docrawl_client.py

```diff
@@ -90,15 +94,15 @@ def _wait_until_page_is_loaded(self, timeout=20):
             except:
                 is_page_loaded = False
             time.sleep(0.5)
-            docrawl_logger.info('Page is still loading, waiting 0.5 sec ...')
+            # docrawl_logger.info('Page is still loading, waiting 0.5 sec ...')
 
         if is_page_loaded:
-            docrawl_logger.success('Page loaded')
+            docrawl_logger.warning(f'Page loaded: {self.get_browser_meta_data()["request"]["url"]}')
```

Comment:
might fail on KeyError -> it's safer to wrap this warning in try: except / if conditions, change the last dictionary to .get()

Reply:
It should not fail on `KeyError` - `self.get_browser_meta_data()['request']['loaded']` should always have some value assigned. If it fails, that means our docrawl has entered some inconsistent state which will lead to further bugs - in that case its due to our programming bugs - which we should solve asap and not silence.

Reply:
That'S true, however I am afraid it will happen that some inconsitent cases can happen in production e.g. I restart redis and its empty, I restart DB and it's not in the expected application state etc. -> then if this error handling is missing it's increasing time for me to deploy, since I will need to implement it again and will take several hours before I spot the issue